std/shared/src/main/scala/cats/effect/std/Hotswap.scala

```diff
@@ -51,6 +50,7 @@ import cats.syntax.all._
  *
  * Ported from https://github.com/typelevel/fs2.
  */
+@deprecated("Use Hotswap2", "3.6.3")
```

Comment:
This will land in 3.7.0, since we only introduce new APIs in minors (and reserve patch versions for bug fixes only).

```suggestion
@deprecated("Use Hotswap2", "3.7.0")
```

Reply:
Fixed, here and elsewhere 👍🏻